src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Heading from "../Common/Heading/Heading";
+import Description from "../Common/Description/Description";
+import Snippet from "../Common/Snippet/Snippet";
+import styles from './modal.module.css'
+
+const Modal = () => {
+    return (
+        <div className={styles.modal_main}>
+            <Heading text="Modal" />
+            <Description text="The modal component provides a solid foundation for creating dialogs, popovers, lightboxes." />
+            <div className={styles.content_block}>
+            <a href="#myModal"><button id='open_modal' className={`${styles.btn} ${styles.btn_primary}`}>Modal</button></a>
+
+                {/* <div id="myModal" className={styles.modal}>
+                    <div className={styles.modal_content}>
+                        <span id='modal_close' className={styles.modal_close}>&times;</span>
+                        <p>Some text in the Modal..</p>
+                    </div>
+                </div> */}
+
+                {/* <a href="#openModal-about">Modal</a> */}
+
+                <div id="myModal" className={styles.modal}>
+                    <div className={styles.modal_content}>
+                        <a href="#close" title="Close" className={styles.modal_close}>X</a>
```

Comment:
The close button (X) is not centered, would recommend making it center.
Also, try bringing it inside the modal and giving it a squircle on hover (no border style without hover).
However, I don't feel strongly about the second point so it's up to you.

Reply:
I have changed the style of close button